indexer-opts/README.md

```diff
@@ -0,0 +1,52 @@
+# indexer-opts - Opts struct for CLI arguments for microindexers
+
+This lib is a shared `clap::Parser` instance that defines CLI arguments for all microindexers.
+
+### Features:
+
+- Basic arguments for indexer to start
+- `init_tracing` for initializing the logs for indexer
+- `StartMode` that allows to choose the strategy for restarts:
+  - `from-latest`
+  - `from-interruption` (default)
+
+### Parameters
+
+- `indexer-id` | **Required** Sets the micro-indexer instance ID (for reading/writing indexer meta-data)
+- `start-block-height` | **Required** Block height to start the stream from (not used if `start_mode == from-latest`)
+- `near-archival-rpc-url` | **Required** NEAR JSON RPC URL
```

Comment:
Yeah but it is required for `from-latest`, otherwise we can't start it from latest, we might rename it to just `rpc-url` for the sake, what do you think?

Reply:
I can propose:

1. Rename it to `rpc-url`
2. Make it optional
3. Throw an error if you try using `from-latest` without `rpc-url` provided

Agree?

Reply:
Ah I see. Agree, let's make it required.
Yeah, let's rename it

Note to future myself: we will also need to add some words to indexer readmes saying where RPC is required to be archival.